modules/Bio/EnsEMBL/Compara/RunnableDB/NewPerSpeciesComparaDB.pm

```diff
@@ -0,0 +1,99 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB
+
+=head1 DESCRIPTION
+
+Creates new MySQL compara database for a single genome_name
+E.g. standaloneJob.pl Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB \
+    -homology_host mysql-ens-compara-prod-2 \
+    -genome_name canis_lupus_familiaris \
+    -curr_release 103
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::NewPerSpeciesComparaDB;
+
+use warnings;
+use strict;
+
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::Compara::Utils::Database qw/ table_exists /;
+use Data::Dumper;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+sub param_defaults {
+    my $self = shift;
+
+    return {
+        %{$self->SUPER::param_defaults()},
+        'homology_host'  => 'mysql-ens-compara-prod-2',
+    };
+}
+
+sub fetch_input {
+    my $self = shift;
+
+    my $release     = $self->param_required( 'curr_release' );
+    my $genome_name = $self->param_required( 'genome_name' );
+    my $host        = $self->param_required( 'homology_host' );
+    # Keep input to fewer parameters
+    my $port     = Bio::EnsEMBL::Compara::Utils::Registry::get_port( $host );
+    my $rw_user  = Bio::EnsEMBL::Compara::Utils::Registry::get_rw_user( $host );
+    my $password = Bio::EnsEMBL::Compara::Utils::Registry::get_rw_pass( $host );
+
+    my $server_uri  = "mysql://" . $rw_user . ":" . $password . "@" . $host . ":" . $port;
+    my $new_db_name = $genome_name . "_" . "compara" . "_" . $release;
```

Comment:
```suggestion
    my $new_db_name = $genome_name . "_compara_" . $release;
```
Just a note: since there won't be divisions any more, I assume there will be a single release number for everything? (just to avoid nasty issues like having 2 release numbers for non-vertebrates)

Reply:
I am not altogether sure what is happening with $release - but in this case it is based on software_version() or reference database version for now rather than a production release exactly. I imagine at some point we will swap it out for a rapid release number instead (?) - yeah really don't know :shrug: